### PR TITLE
Add Reference section

### PR DESCRIPTION
**Pull request by rudri182**
I have added some links for copied text for more reference.

### CODE REVIEW

**Comment by rudri182** [comment]
@alodahl please check if it is alright now or not. The PR is #683 from which I have made these changes.

**Comment by rudri182** [comment]
I have removed the reference section and put the reference where the text is located in the outline. Is it proper or anything is missing? 

One flaw in the outline is the image in the starting is not working, it is showing error 404.

File: roles-in-tech/security.md [hunk 1]
```diff
@@ -12,7 +12,7 @@
 - [Intro to Security](/security/web-security.md)
 
 ### Motivation
-- Computer security is important, primarily to *keep your information protected*. It’s also important for your computer’s overall health, helping to prevent viruses and malware and allowing programs to run more smoothly.
+- Computer security is important, primarily to *keep your information protected*. It’s also important for your computer’s overall health, helping to prevent viruses and malware and allowing programs to run more smoothly. Refer more at [The Importance of Computer Security](https://blog.onsharp.com/the-importance-of-computer-security)
```
Comment (alodahl) [inline roles-in-tech/security.md]: Refer more at [The Importance of Computer Security](https://blog.onsharp.com/the-importance-of-computer-security)

Change to just:
([The Importance of Computer Security](https://blog.onsharp.com/the-importance-of-computer-security))

File: roles-in-tech/security.md [hunk 2]
```diff
@@ -95,7 +95,7 @@ These are some misconceptions that people have in their mind but it is not alway
 
 - **Cyber Security is just a form of defense** - “The best defense is a good offense.”
 
-- **You will never be attacked** - Unfortunately, the attacked business is often the last to know that it's been breached. It's only after banks discover fraudulent charges that businesses hear about a suspected hack.
+- **You will never be attacked** - Unfortunately, the attacked business is often the last to know that it's been breached. It's only after banks discover fraudulent charges that businesses hear about a suspected hack. More understanding at [Top 8 Data Breach Misconceptions](https://www.insureon.com/blog/post/2015/01/12/top-8-data-breach-misconceptions.aspx)
```
Comment (alodahl) [inline roles-in-tech/security.md]: Change to just:

([Top 8 Data Breach Misconceptions](https://www.insureon.com/blog/post/2015/01/12/top-8-data-breach-misconceptions.aspx))

**Comment by rudri182** [comment]
Changes are done 👍 Please review it.

**Comment by rudri182** [comment]
Are there any more changes to be done? @alodahl

File: roles-in-tech/security.md [hunk 3]
```diff
@@ -107,6 +107,9 @@ These are some misconceptions that people have in their mind but it is not alway
```
Comment (alodahl) [inline roles-in-tech/security.md]: image

**Comment by rudri182** [comment]
I have verified all the sections. I think now it is alright now. If there are still any changes or links to be added then, please draw my attention to it.

File: roles-in-tech/security.md [hunk 4]
```diff
@@ -95,18 +97,21 @@ These are some misconceptions that people have in their mind but it is not alway
...
+- **Emails are secure** - Over 400 businesses are targeted daily, and these emails are growing increasingly and it is challenging to identify since they’re designed to mimic legitimate communication so they can gain access to your sensitive information – including usernames and passwords.([Cybersecurity Misconceptions and Challenges Faced by Small Businesses](https://datafloq.com/read/cybersecurity-misconceptions-challenges-faced-sme/5467))
 
 - **Small organizations will not be attacked** - Major organizations assume that they are small and will not be targeted as we are a small company. The research shows that 70% of cyber attacks target small businesses. So today for everyone it is very important to take care of its security if its data is worth value for money, whether it is a website for fun or it is a huge organization.
 
+- ([9 Steps to Land a Cyber Security Job, According to Experts](https://www.topuniversities.com/student-info/careers-advice/9-steps-land-cyber-security-job-according-experts))
```
Comment (alodahl) [inline roles-in-tech/security.md]: Is this link citing line 110? If so, please move it up next to the quote like your other bullet-points.

Comment (rudri182) [reply roles-in-tech/security.md]: I have made changes accordingly. Please review it.

**Pull request merged**